config/local-development/tilt/env-replace-image.yaml

```diff
@@ -1,4 +1,16 @@
-- op: replace
-  path: /spec/template/spec/containers/1/image
-  value:
-    quay.io/$repo/vault-config-operator:latest
+apiVersion: apps/v1
```

Comment:
mm ok this is new and I am confused. maybe we can meet to discuss this

Reply:
sure